docker/supervisor-celery.conf

```diff
@@ -2,7 +2,13 @@
 logfile=/code/logs/supervisord-celery.log
 
 [program:olympia]
-command=celery worker --app=olympia
+# This command mimics how we run it in prod from puppet:
+# https://github.com/mozilla-services/puppet-config/blob/master/amo/modules/olympia/manifests/celery.pp#L16-L24
```

Comment:
you might want a commit hash in the url? So the link stays relevant even if the file changes?